assets/css/main.css

```diff
@@ -2,623 +2,650 @@ html, body {
+  width:49.5%;
+  margin:0;
+}
+#what-were-building .project-profile:first-child{
+  border-right:1px solid rgba(0,0,0,.08);
+  width:49.5%;
+}
+#what-were-building .project-profile img{
+  width:200px;
+  max-width:80%;
+  margin:0 auto;
 }
 
-#who-we-are {
-  margin-top: 100px;
+#what-were-building .project-profile h4{
+  margin:15px 0 5px 0;
+  padding:0 25px;
+  color:#555;
 }
+#what-were-building .project-profile h5{
+  margin:0 0 25px 0;
+  padding:0 50px;
+  color:#555;
 
-#hire-us h2 {
-  margin-top: 0;
+}
+#what-were-building .project-profile p{
+  font-size:16px;
+  line-height:24px;
+  margin:0 auto 0px;
+  color:#555;
+  width:300px;
+  max-width:100%;
 }
 
-.team-photo {
-  height: 150px;
-  width: 150px;
-  float: left;
-  margin: 8px 10px 0 0;
+#what-were-building .project-profile .btn{
+  color:rgba(225, 73, 38, 1);
+  font-size:16px;
+  text-decoration:none;
+  letter-spacing:.05em;
+  border-bottom:1px solid rgba(225, 73, 38, .2);
+  padding-bottom:2px;
 }
 
-.team-list-item-description {
-  margin: 0 25px 0 0;
-  line-height: 1.3rem;
+.logo-container{
+  display:block;
+  height:100px;
+  width:350px;
+  padding: 10px 0;
+  max-width:100%;
+  text-align:center;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -webkit-align-items: center;
+  -ms-flex-align: center;
+  align-items: center;
+  margin:0 auto;
+  background:rgba(255,255,255,1);
+  margin-bottom:20px;
 }
 
-@media screen and (max-width: 850px) {
-  #who-we-are .component-list-item {
-    width: 100%;
-    padding: 0;
-    margin: 0 0 30px 0;
-  }
+/*CTA BLOCK*/
+#cta-block.wide-block{
+  background:rgba(24,34,41, 1);
+  color:#fff1ef;
+}
+#cta-block .inner-wrapper h3{
+  margin-bottom:15px;
+  color:#fff;
+}
+#cta-block .inner-wrapper p{
+  margin:0 auto 35px auto;padding:0;
+  line-height:100%;
+}
 
-  #who-we-are .team-photo {
-    margin: 3px 15px 0 0;
-  }
+/*FOOTER*/
+footer{
+  background:rgba(24,34,41, 1);
+  border-top:1px solid rgba(82,134,142,.3);
+  color:#52868e;
+  padding:0;
+  width:1000px;
+  margin:0 auto;
+  text-align:center;
 }
 
+#footer-wrapper{
+  background:rgba(24,34,41, 1);
+}
 
-@media screen and (max-width: 650px) {
-  #who-we-are .component-list-item {
-    margin: 10px 0 30px 0;
-  }
+footer ul{
+  margin: 35px 0 20px 0;padding:0;
+  list-style:none;
+}
 
-  #who-we-are .team-photo {
-    float: none;
-    margin: 0 0 15px 0;
-  }
+footer ul.social li{
+  margin:0;padding:0;
+  display:inline-block;
 }
 
-.component-list-wrapper {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  -webkit-flex-wrap: wrap;
-      -ms-flex-wrap: wrap;
-          flex-wrap: wrap;
-  -webkit-box-pack: justify;
-    -webkit-justify-content: space-between;
-        -ms-flex-pack: justify;
-            justify-content: space-between
+footer ul.social li a{
```

Comment:
this could just be `.footer a` or if there's other links in there `.footer .social a`